A32NX/html_ui/Pages/VCockpit/Instruments/Airliners/A320_Neo/PFD/A320_Neo_PFD.js

```diff
@@ -111,10 +114,27 @@ class A320_Neo_PFD_MainPage extends NavSystemPage {
             this.vsFlash.setAttribute("visibility", "hidden");
         }
 
-        var ADIRSState = SimVar.GetSimVarValue("L:A320_Neo_ADIRS_STATE", "Enum");
-        var PFDAlignedFirst = SimVar.GetSimVarValue("L:A32NX_ADIRS_PFD_ALIGNED_FIRST", "Bool");
+        let IsOnGround = SimVar.GetSimVarValue("SIM ON GROUND", "Bool");
         var PFDAlignedATT = SimVar.GetSimVarValue("L:A32NX_ADIRS_PFD_ALIGNED_ATT", "Bool");
 
+        if (IsOnGround && PFDAlignedATT) {
+            this.groundCursor.style.display = "block";
+            this.groundCursorLimits.style.display = "block";
+        }
+        else {
```

Comment:
Please put the `else` on the same line as the closing bracket of the `if` block, like `} else {`.